pkg/klusterlet/clusterinfo/distributionInfo_sync.go

```diff
@@ -25,7 +25,7 @@ type distributionInfoSyncer struct {
 func (s *distributionInfoSyncer) sync(ctx context.Context, clusterInfo *clusterv1beta1.ManagedClusterInfo) error {
 	// currently we only support OCP in DistributionInfo
 	switch clusterInfo.Status.KubeVendor {
-	case clusterv1beta1.KubeVendorOpenShift:
+	case clusterv1beta1.KubeVendorOpenShift, clusterv1beta1.KubeVendorOSD:
```

Comment:
not only OCP and OSD, but also ARO, ROSA, ROKS which are in OCPList

Reply:
That's  a fun truth, in the `ProductOCPList`, other type will finally be represent by `KubeVendorOpenShift`: https://github.com/stolostron/multicloud-operators-foundation/blob/b4de56dec0b68530ad4701805403e2b66272e484/pkg/klusterlet/clusterinfo/defaultInfo_sync.go#L88